src/lib.rs

```diff
@@ -770,6 +771,43 @@ impl <'a> BitXorAssign for FixedBitSet
     }
 }
 
+/// Two `FixedBitSet`s are equal if and only if they have the exact same set of "one" elements
+impl PartialEq for FixedBitSet
+{
+    #[inline]
+    fn eq(&self, other: &Self) -> bool {
+        self.ones().eq(other.ones())
```

Comment:
While conceptually this algorithm for comparison might be good, I'm afraid that it is slow. Comparison should be based on comparing the blocks, I would guess.

Reply:
It makes sense!

I've rewrote the code to extract the trimmed slice of blocks instead of using the iterator of ones.